mne_sandbox/connectivity/cfc.py

```diff
@@ -332,16 +428,78 @@ def _phase_amplitude_coupling(data, sfreq, f_phase, f_amp, ixs,
         return pac, freq_pac
 
 
+def _check_pac_times(events, times, tmin, tmax):
+    # Times and time windows
+    err = 'Must supply one of events or times (not both)'
+    if all(ii is not None for ii in [events, times]):
```

Comment:
actually that doesn't work, but I think `if (events is times is None) or (events is not None and times is not None)` does